src/nodetree.rs

```diff
@@ -229,15 +265,19 @@ impl Node {
 #[pyclass]
 #[derive(Debug, Clone, Default, PartialEq, Eq)]
 pub struct CallTreeFunction {
+    pub parent_fingerprint: Option<u32>,
+    pub stack_fingerprint: Option<u32>,
     pub fingerprint: u32,
     pub function: String,
     pub package: String,
     pub in_app: bool,
     pub self_times_ns: Vec<u64>,
+    pub total_times_ns: Vec<u64>,
```

Comment:
Not sure why you mixed in the `total_times_ns` vec in this PR but it'd be good the `sum_total_time_ns` as an attribute as well.